### PR TITLE
likes and comments on posted/shared articles notifications

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/PersonalApplications.rst [hunk 1]
```diff
@@ -1285,9 +1285,33 @@ You will receive the **My News shared** notification when a user shares one of y
 
 	|image99|
 
-**My News liked**
+**My posted News liked**
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst:1288]: "My liked posted News" ?

File: docs/PersonalApplications.rst [hunk 2]
```diff
@@ -1285,9 +1285,33 @@ You will receive the **My News shared** notification when a user shares one of y
...
+
+You will receive the **My posted News liked** notification when a user likes one of your articles.
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive the **My posted News liked** notification when a user liked one of your articles.
```

File: docs/PersonalApplications.rst [hunk 3]
```diff
@@ -1285,9 +1285,33 @@ You will receive the **My News shared** notification when a user shares one of y
...
+
+	|image100|
+
+	
+**My shared News liked**
+
+You will receive the **My shared News liked** notification when a user likes an article you shared.
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive the **My shared News liked** notification when a user liked an article you shared.
```

File: docs/PersonalApplications.rst [hunk 4]
```diff
@@ -1285,9 +1285,33 @@ You will receive the **My News shared** notification when a user shares one of y
...
+
+	|image101|
+
+
+**My posted News commented**
+
+You will receive the **My posted News commented** notification when a user comments one of your articles.
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive the **My posted News commented** notification when a user commented one of your articles.
```

File: docs/PersonalApplications.rst [hunk 5]
```diff
@@ -1285,9 +1285,33 @@ You will receive the **My News shared** notification when a user shares one of y
...
+
+	|image102|
+	
+
+**My shared News commented**
+
+You will receive the **My shared News commented** notification when a user comments an article you shared.
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive the **My shared News commented** notification when a user commented an article you shared.
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/PersonalApplications.rst (hunk 1 above)
Comment (plamarque) [reply docs/PersonalApplications.rst:1288]: or "My liked news"

File: docs/PersonalApplications.rst (hunk 2 above)
Comment (plamarque) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive this notification when a user likes one of your articles.
```

File: docs/PersonalApplications.rst (hunk 3 above)
Comment (plamarque) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive this notification when a user liked an article you shared.
```

File: docs/PersonalApplications.rst (hunk 5 above)
Comment (plamarque) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive this notification when a user commented an article you shared.
```

**Review by plamarque — CHANGES_REQUESTED** [review]

File: docs/PersonalApplications.rst (hunk 1 above)
Comment (samehaidi) [reply docs/PersonalApplications.rst:1288]: Shall we put the exact text we have in the page notifications settings? @plamarque @thomasdelhomenie

Comment (plamarque) [reply docs/PersonalApplications.rst:1288]: well, make the change in product :p

**Review by plamarque — APPROVED** [review]

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**